### ISSUE TITLE
import2

### ISSUE THREAD

**Issue by THESAURABH07** [THESAURABH07/data-main#2]
3.Install and set up scikit-learn and other necessary tools.

from sklearn.preprocessing import StandardScaler
from sklearn.preprocessing import MinMaxScaler
from sklearn.preprocessing import RobustScaler
from sklearn.preprocessing import Normalizer
import numpy as np
import pandas as pd

data={"speed

":[10,20,30,40,50],"distance":[1000,2000,3000,4000,5000]}
print(data)
df=pd.DataFrame(data)
print(df)

ss=StandardScaler()
x1=ss.fit_transform(df)
print(x1)

mm=MinMaxScaler()
x2=mm.fit_transform(df)
print(x2)

rs=RobustScaler()
x3=rs.fit_transform(df)
print(x3)

n=Normalizer()
x4=n.fit_transform(df)
print(x4)

Ml_pgm_4
# Write a program to Load and explore the dataset of .CVS and excel files using pandas.
import pandas as pd

df=pd.read_csv(r"path to csv file")
#install excel  package  python -m pip install openpyxl
df1=pd.read_excel(r"path to excel file")
pd.set_option(“display.expand_frame_repr”,False)
print(df)
print(df1)
print("columns of df")
print(df.columns)
print("columns of df1")
print(df1.columns)
print("describe of df")
print(df.describe())
print("describe of df1")
print(df1.describe())
print("head of df")
print(df.head)
print("head of df1")
print(df1.head)
